Automattic/pocket-casts-android
# Transcripts - Show scrollbar

## Description
This displays a scrollbar on the transcript view.

I added a basic implementation without any drag capabilities (p1722425522642369-slack-C078746V6UX). Note that scrollbars are in the [Compose roadmap](https://developer.android.com/jetpack/androidx/compose-roadmap?authuser=1#:~:text=headers%20for%20lists-,Scrollbars,-Flow%20layouts%20new), so hopefully we'll have a fully functional scrollbar soon. 

## Testing Instructions
1. Open transcript view with scrollable text (e.g. episode from podcast: "Cautionary Tales" (https://pcast.pocketcasts.net/8zpmjiru))
2. Try to scroll the view
3. Notice that scrollbars are visible.
** Test for a transcript with HTML mime-type (eg: episode from podcast `If books could kill` https://pca.st/tqib657g also, as those are displayed on Android TextView.


## Screenshots or Screencast 

https://github.com/user-attachments/assets/db4b766f-d5f8-4466-8edf-36253a8b4b27


## Checklist
 
#### I have tested any UI changes...
